fastapi_sqla/asyncio_support.py

```diff
@@ -148,15 +149,15 @@ async def paginate_query(
     limit: int,
     *,
     scalars: bool = True,
-) -> Page[T]:
+) -> Page:
```

Comment:
It didn't made a lot of sense to return `[T]` here since the TypeVar wasn't bound to anything in that context. It would have made sense if for example we had the signature like:

```
async def paginate_query(
    query: Select[T],
    session: AsyncSession,
    total_items: int,
    offset: int,
    limit: int,
    *,
    scalars: bool = True,
) -> Page[T]:
```